drivers/python/rethinkdb/_import.py

```diff
@@ -515,9 +523,9 @@ def csv_reader(task_queue, filename, db, table, options, progress_info, exit_eve
 
 # This function is called through rdb_call_wrapper, which will reattempt if a connection
 # error occurs.  Progress will resume where it left off.
-def create_table(progress, conn, db, table, pkey, sindexes):
+def create_table(progress, conn, db, table, pkey, sindexes, shards, replicas):
     if table not in r.db(db).table_list().run(conn):
-        r.db(db).table_create(table, primary_key=pkey).run(conn)
+        r.db(db).table_create(table, primary_key=pkey, shards=shards, replicas=replicas).run(conn)
```

Comment:
I don't think this does what you intend when `shards` or `replicas` is `None`:

```py
rethinkdb.errors.RqlRuntimeError: Expected type NUMBER but found NULL. in:
r.db('test').table_create('foo', shards=None, replicas=None)
                                        ^^^^
```

It might be easier to pass down some `kwargs` `dict` that you could expand with `table_create(table, **kwargs)`, but that gets complicated a bit by `primary_key`.